integration-tests/example.env

```diff
@@ -15,11 +15,34 @@ export SLACK_USER="U000000000" # User ID of the person running the soak tests to
 
 ########## Network Settings ##########
 
-# Private values that will be retrieved when running on non-simulated networks
-export EVM_PRIVATE_KEYS="private,funding,keys" # Comma-separated list of private keys to use when running on live networks
+# Select a pre-defined network 
+export SELECTED_NETWORK="Goerli"
```

Comment:
If the test needs to handle more than one network, would it make sense to make it a list of networks?

Reply:
Use case being something like "I want to run OCR on Goerli and Optimism"?

Reply:
Something like CCIP. Or probably the case where we make use of EVM_NODEs instead of EVM_CHAIN_ID